src/api/projects.js

```diff
@@ -1,60 +1,99 @@
 const express = require('express');
 const projetsExample = require('./dev/projectsExamples.json');
+const prisma = require('../../prismaClient');
 
 const router = express.Router();
 
 router.get('/', (req, res) => {
-  res.status(200).json(projetsExample);
+  prisma.project
+    .findMany()
+    .then((projects) => {
+      res.status(200).json(projects);
+    })
+    .catch((error) => {
+      res.status(500).json(error);
+    });
 });
 
-router.get('/:id', (req, res) => {
-  const projectID = +req.params.id;
-
-  const result = projetsExample.find((project) => project.id === projectID);
-
-  if (result) {
-    res.status(200).json(result);
-  } else {
-    res.status(404).json({ message: 'project not found' });
+router.get('/:id', async (req, res) => {
+  const { id } = req.params;
+  try {
+    const project = await prisma.project.findUnique({
+      where: {
+        id,
+      },
+    });
+    res.status(200).json(project);
+  } catch (err) {
+    res.status(500).json(err);
```

Comment:
500 HTTP codes are using to server errors. 500 is "internal server error". Please use another code.